File: javascript/programmers/javascript/level1/2016년.js
const a = 5;
const b = 24;

function solution(a, b) {
  const days = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
  const DayOfWeek = ["SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"];
  let answer = "";
  let cal_days = 0;
  for (let i = 0; i < a - 1; i++) {
    cal_days += days[i];
  }
  cal_days += b;
  answer = DayOfWeek[(cal_days + 4) % 7];
  return answer;
}

console.log(solution(a, b));

File: python/programmers/level1/2016년.py
def solution(a, b):
    day = ['SUN', 'MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT']
    dayOfMonth = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
    ans = 4
    for i in range(a - 1):
        ans += dayOfMonth[i]
    ans += b
    answer = day[ans % 7]

    return answer


print(solution(5, 24))
